server/speak.py

```diff
@@ -5,9 +5,7 @@
 FILENAME = "tts.mp3"
 
 def get_speak (text, lang):
-    print("speak" + text)
-    print("speaklang" + lang)
     tts = gTTS(text=text, lang=lang, slow=False)
     tts.save(FILENAME)
 
-mp3_path = os.path.join(os.path.dirname(__file__), "tts.mp3")
\ No newline at end of file
+mp3_path = os.path.join(os.path.dirname(__file__), "../static/tts.mp3")
```

Comment:
Just noticed the `static` folder is pointing to `../static/dist/` instead. Usually we create a file named `constants.py` that contains constant variables that will be used often throughout the code-base for example: `STATIC_PATH = "static/dist"`. So when you'll have to update it, you will only update in one place and such errors will be less likely to occur.

- A nitpick 😆 make sure to add a newline after this one, Github don't like ending files without an empty line:

![image](https://user-images.githubusercontent.com/26286907/79283125-8dbc4280-7ebf-11ea-93f4-43bfa5e70ca3.png)

Reply:
Is that a bad thing ?
At first when i had only "tts.mp3" it was creating the file on the static folder, when i changed to "static/tts.mp3" it kept on creating the file on that folder too. 
Should i keep this way or change it ?

Reply:
- You mean having a nested folder inside `static` is bad ? no, not at all.
- I think we want it to  be created as `static/dist/tts.mp3` instead of `static/tts.mp3`

Don't mind the `constants.py` suggestion for now, let's leave it for later 😅

Reply:
constants.py is a great sugestion for big projects, my newbie project only has one variable.. :)